torch/testing/_internal/common_nn.py

```diff
@@ -5188,7 +5215,7 @@ def apply_fn(input, target, *params):
         else:
             inputs = input + params + (target,)
 
-            def apply_fn(input1, input2, target, *params):
+            def apply_fn(input1, input2, target, *params):  # type: ignore[misc]
```

Comment:
yep, this one is best ignored, too much code duplication to make `mypy` happy here